CMakeLists.txt

```diff
@@ -87,7 +87,7 @@ find_program(KCONFIGLIB olddefconfig)
 if(NOT KCONFIGLIB)
   message(
     FATAL_ERROR "Kconfig environment depends on kconfiglib, Please install:
-  $ pip install kconfiglib")
+  $ sudo apt install python3-kconfiglib")
```

Comment:
Does this work if your computer is not running Debian or another `apt`-based system?

Reply:
Ah good point, I guess I will either specify or revert this change what do you think?

Reply:
Perhaps it can be a recommendation. For example:

```
 message(
    FATAL_ERROR "Kconfig environment depends on kconfiglib. Please install.
  If using pip:
    $ pip install kconfiglib
  If using another package manager, such as apt:
    $ sudo apt install python3-kconfiglib
  ")
```

Thoughts?

Reply:
Seems super clear. I'll modify it when I have my pc :)

Reply:
Great! Thanks!